voltdb/src/main/java/org/voltdb/sortedvolttable/VoltDBTableSortedMergeWrangler.java

```diff
@@ -0,0 +1,207 @@
+package org.voltdb.sortedvolttable;
```

Comment:
this should be in `com.yahoo.ycsb.db.voltdb` or a package under that

Reply:
Should be fixed in 0a534e8434d7e854901fbcf825df0403cc985ba3 .